### PR TITLE
feat(helmfile): add spec

### PR DESCRIPTION
**Pull request by Wail-Mohammed**
**What kind of change does this PR introduce? (Bug fix, feature, docs update, ...)**
Feature (Spec feature added for helmfile)
**What is the current behavior? (You can also link to an open issue here)**
Closes https://github.com/withfig/autocomplete/issues/1201 
**What is the new behavior (if this is a feature change)?**
Fig autocomplete can now auto complete helmfile commands and flags.
**Additional info:**
This PR is aiming to close issue # 1201 by adding helmfile.ts to the src folder in order to add helmfile commands to autocomplete.

### CODE REVIEW

File: src/helmfile.ts [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+        "DEPRECATED: sync releases from state file (helm upgrade --install)",
+    },
+    {
+      name: "completion",
+      description: "Generate the autocompletion script for the specified shell",
+    },
+    {
+      name: "delete",
+      deprecated: true,
+      description: "DEPRECATED: delete releases from state file (helm delete)",
```
Comment (mschrage) [inline src/helmfile.ts:31]: Please use the `[deprecated](https://fig.io/docs/reference/suggestion#deprecated)` field to signify this type of information.

**Comment by Wail-Mohammed** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/helmfile.ts [hunk 2]
```diff
@@ -0,0 +1,224 @@
+const completionSpec: Fig.Spec = {
+  name: "helmfile",
+  description: "",
```
Comment (fedeci) [inline src/helmfile.ts:3]:
```suggestion
  description: "Deploy helm charts.",
```

**Review by fedeci — CHANGES_REQUESTED** [review]

File: src/helmfile.ts [hunk 3]
```diff
@@ -0,0 +1,224 @@
...
+      deprecated: true,
+      description:
+        "DEPRECATED: sync releases from state file (helm upgrade --install)",
+    },
+    {
+      name: "completion",
+      description: "Generate the autocompletion script for the specified shell",
+    },
+    {
+    {
```
Comment (fedeci) [inline src/helmfile.ts:29]:
```suggestion
```

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Thanks!

**Pull request merged**